engine/lite/components/kafka/src/test/scala/pl/touk/nussknacker/engine/lite/components/LiteKafkaAvroFunctionalTest.scala

```diff
@@ -94,4 +52,473 @@ class LiteKafkaAvroFunctionalTest extends FunSuite with Matchers with ValidatedV
 
     new LiteKafkaTestScenarioRunner(mockSchemaRegistryClient, mockedComponents, config)
   }
+
+  test("should test end to end kafka avro record data at sink / source") {
```

Comment:
Do we have some sanity check for "for all avro schemas if we do #input => #input with same schemas on input and output, it will work"?

Reply:
I've added two property based test for #input -> #input, and some SpEL as output.